### ISSUE TITLE
Performance numbers for 4bit non-uniform quantized resnet20 on Cifar10

### ISSUE THREAD

**Issue by rapwag01** [Tencent/PocketFlow#207]
This template is for miscellaneous issues not covered by the other issue categories.

For questions on how to work with PocketFlow, or support for problems that are not verified bugs in PocketFlow, please go to [StackOverflow](https://stackoverflow.com).

For high-level discussions about TensorFlow, please post to [discuss group](https://groups.google.com/forum/#!forum/pocketflow).

**Comment by rapwag01** [comment]
Dear Pocketflow Team,

I'm running below command after having downloaded the pre-trained model from [here](https://api.ai.tencent.com/pocketflow/models_resnet_20_at_cifar_10.tar.gz) 

./scripts/run_local.sh nets/resnet_at_cifar10_run.py --learner=non-uniform

I have also adjusted the learning rate decay to 
decay_rates = [1e-3, 1e-4, 1e-5] and nuql_quant_epochs=120

Using above settings the quantized model achieves 86.37% accuracy (note that for default settings accuracy is only at 57%). I have tried different combinations of hyper-parameters but cannot recover the accuracy of the pre-trained model, which is 91.31%.

Have you managed to achieve >90% accuracy for a non-uniform quantized resnet20 model on Cifar10?

Thanks in advance for any tips & tricks!

**Comment by rapwag01** [comment]
solved - this issue can be closed.

./scripts/run_local.sh nets/resnet_at_cifar10_run.py --learner=non-uniform **--enbl_warm_start** 
with default settings gives 91.32% in top1 accuracy.

Please update the documentation, it is unclear that the warm-start switch has to be provided!

**Comment by huxianer** [comment]
@jiaxiang-wu Could you tell me where is the script to test the accuracy and flops of the model which is  
compressed?

**Comment by jiaxiang-wu** [comment]
@huxianer 
To test the accuracy of *.ckpt models, you can use the same execution script as training and pass `--exec_mode=eval` to performance evaluation. We do not have evaluation tools for *.pb & *.tflite models.
To calculate the FLOPs, you may use TensorFlow's tf.profiler package:
https://stackoverflow.com/questions/45085938/tensorflow-is-there-a-way-to-measure-flops-for-a-model

**Comment by huxianer** [comment]
@jiaxiang-wu Thank you!
